formatters.py

```diff
@@ -197,7 +197,7 @@ def _mk_synopsis(self, parser):
 
     def _mk_title(self, prog):
         # and an easy to use reference point
-        title = ".. _man_%s:\n\n" % prog
+        title = ".. _man_%s:\n\n" % prog.replace(' ', '-')
```

Comment:
duplication is evil and we fell for it -- we should get some proper datalad_buildsupport python package/module to avoid duplicating across all extensions etc... filed https://github.com/datalad/datalad/issues/5719

meanwhile -- this one would be the same as in https://github.com/datalad/datalad-extension-template/blob/master/_datalad_buildsupport/formatters.py#L204  so  -- good